### PR TITLE
Add "code" field

### PR DESCRIPTION
**Pull request by stefanvit**
## Description
Add "code" field

**Resolves 

## Review Time Estimate
- [ ] Immediately
- [ ] Within a week
- [x] When possible

## Type of changes
- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [x] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: driver/web/conversions_application.go [hunk 1]
```diff
@@ -20,11 +20,25 @@ import (
 	"core-building-block/utils"
 )
 
+// Apps
+func appToDef(item model.Application) Def.Apps {
+
+	return Def.Apps{Id: &item.ID, Name: item.Name, Code: item.Code}
```
Comment (petyos) [inline driver/web/conversions_application.go:26]: The name should be Def.App but not Def.Apps as this is single entity

File: driver/web/docs/gen/def.yaml [hunk 2]
```diff
@@ -6232,6 +6235,20 @@ components:
           type: boolean
         time_since_login_period:
           type: integer
+    Apps:
```
Comment (petyos) [inline driver/web/docs/gen/def.yaml:6238]: This is not Apps but App

File: driver/web/docs/gen/def.yaml [hunk 3]
```diff
@@ -7099,7 +7120,7 @@ components:
       type: string
       description: |
         Auth login creds for auth_type="oidc" (or variants)
-          - full redirect URI received from OIDC provider
```
Comment (petyos) [inline driver/web/docs/gen/def.yaml:7102]: Why has this been changed?

File: driver/web/docs/gen/gen_types.go [hunk 4]
```diff
@@ -1,12 +1,12 @@
 // Package Def provides primitives to interact with the openapi HTTP API.
 //
-// Code generated by github.com/deepmap/oapi-codegen version v1.16.2 DO NOT EDIT.
```
Comment (petyos) [inline driver/web/docs/gen/gen_types.go:3]: Please update the to latest tool version and then generate the struct. You use currently old version.

File: go.mod [hunk 5]
```diff
@@ -40,50 +40,101 @@ require (
 )
 
 require (
+	github.com/BurntSushi/toml v1.3.2 // indirect
```
Comment (petyos) [inline go.mod:43]: You issue does not suppose any changes in the dependencies, so go.mod and go.sum should not be modified.

**Review by petyos — CHANGES_REQUESTED** [review]

File: go.mod (hunk 5 above)
Comment (stefanvit) [reply go.mod:43]: this was generated from go.mod

**Pull request merged**